### PR TITLE
[Draft] Adds `TakeScreenshot` command

### PR DESCRIPTION
**Pull request by ferrolho**
Hi, @rdeits! :slightly_smiling_face: Hope you are doing well. I have taken the initiative to implement #61. This is just a proposal, so feel free to go over my changes and review them as you wish. It goes in hand with [rdeits/meshcat#79](https://github.com/rdeits/meshcat/pull/79).

### CODE REVIEW

File: src/visualizer.jl [hunk 1]
```diff
@@ -108,6 +108,9 @@ function update_tree!(core::CoreVisualizer, cmd::SetAnimation, data)
     core.tree.animation = data
 end
 
+function update_tree!(core::CoreVisualizer, cmd::TakeScreenshot, data)
+    core.tree.animation = data
```
Comment (rdeits) [inline src/visualizer.jl:112]: This doesn't look right. The tree update just ensures that any commands used to create the scene will be replayed when a new browser window connects or reloads. But we don't want to re-trigger taking a screenshot, so this particular method shouldn't do anything. I believe it should be sufficient to keep this method but leave the body of it blank (or explicitly put a body of `nothing` to make it more clear that nothing needs to happen).

File: src/visualizer.jl [hunk 2]
```diff
@@ -247,6 +250,16 @@ function setanimation!(vis::Visualizer, anim::Animation; play::Bool=true, repeti
     send(vis.core, cmd)
 end
 
+"""
+Take a screenshot of the current visualizer.
+
+$(TYPEDSIGNATURES)
+"""
+function takescreenshot!(vis::Visualizer)
```
Comment (rdeits) [inline src/visualizer.jl:258]: Perhaps `save_image` as well, to match https://github.com/rdeits/meshcat/pull/79#discussion_r563339743 ?

**Review by rdeits — COMMENTED** [review]
Thanks! This will be a nice feature to have

**Comment by ferrolho** [comment]
Hi, Robin :slightly_smiling_face: I have finished the requested changes.

**Comment by rdeits** [comment]
Thanks again! Could you please also update this line: https://github.com/rdeits/MeshCat.jl/blob/ee5af282c722c36e0083445a86e883325281a206/deps/build.jl#L5 to point to the new `master` commit here: https://github.com/rdeits/meshcat/commit/2355f01a8310314c1d7fc7e51bd01006549beaae

**Pull request merged**